west.yml

```diff
@@ -101,7 +101,7 @@ manifest:
     - name: openthread
       remote: zephyrproject
       path: modules/lib/openthread
-      revision: f460532d4afa5d49feba241e5dc31c56123d31a8
+      revision: pull/43/head
```

Comment:
Perhaps we could add OpenThread back to Zephyrs `name-allowlist` already as we use upstream repo again in NCS? I don't think we need to wait for the upmerge, since we cherry-pick the manifest update to Zephyr anyway.

Reply:
Yes, makes sense.